tasks/clamav-scan.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: clamav-scan
+spec:
+  params:
+    - name: image-digest
+      description: Image digest to scan
+    - name: image-url
+      description: Url to image
+  steps:
+    - name: extract-and-scan-image
+      image: quay.io/redhat-appstudio/hacbs-test:v1.0.1
+      script: |
+        # copy virus database from sidecar
+        cp -r /tmp/clamdb/* /var/lib/clamav
+
+        # strip new-line escape symbol from parameter and save it to variable
+        imageanddigest=$(echo '$(params.image-url)'@'$(params.image-digest)' | tr -d '\n')
```

Comment:
You can also use `echo -n` to not emit a newline

Reply:
Unofrtunately the newline is a part of a parameter: image-url, so it need to be dealth with this way.